examples/dash_display_advancedtest.py

```diff
@@ -30,11 +31,24 @@
 back = touchio.TouchIn(board.CAP7)
 submit = touchio.TouchIn(board.CAP8)
 
-try:
-    from secrets import secrets
-except ImportError:
-    print("WiFi secrets are kept in secrets.py, please add them there!")
-    raise
+# Get wifi details and more from a settings.toml file
+# tokens used by this Demo: CIRCUITPY_WIFI_SSID, CIRCUITPY_WIFI_PASSWORD
+#                           CIRCUITPY_AIO_USERNAME, CIRCUITPY_AIO_KEY
+secrets = {}
+for token in ["ssid", "password"]:
+    if getenv("CIRCUITPY_WIFI_" + token.upper()):
+        secrets[token] = getenv("CIRCUITPY_WIFI_" + token.upper())
+for token in ["aio_username", "aio_key"]:
+    if getenv("CIRCUITPY_" + token.upper()):
+        secrets[token] = getenv("CIRCUITPY_" + token.upper())
```

Comment:
How about just using uppercase tokens?
```suggestion
for token in ["SSID", "PASSWORD"]:
    if getenv("CIRCUITPY_WIFI_" + token):
        secrets[token] = getenv("CIRCUITPY_WIFI_" + token)
for token in ["AIO_USERNAME", "AIO_KEY"]:
    if getenv("CIRCUITPY_" + token):
        secrets[token] = getenv("CIRCUITPY_" + token)
```